### DISCUSSION TITLE
Stop Negative Chart Values

### DISCUSSION THREAD

**Discussion by MarkWildey** [qax-os/excelize#1198] [Q&A] [answered]
Hi
I realise this isn't an excelize issue, but looking for advice from the community.

I use excelize to generate a few charts and in some cases, I get a y-axis showing a negative range even though none of the graphed data contains negative values.

Looking at Google I can see this is possibly an Excel "smoothing" problem.

Although I can specify "minimum" value on the x-axis/y-axis this doesn't influence the generated graph. 

Is there a way, programmatically, I can overcome this problem

Thanks in advance 

![image](https://user-images.githubusercontent.com/24918461/161962654-02d9fa87-d47b-4b30-889f-357abe5eab08.png)

**Comment by xuri** [comment]
Thanks for your feedback. Which version of the library and Excel application are you using? Could you provide generated spreadsheet file attachment without confidential info?

**Comment by MarkWildey** [comment]
Hi
Thank you got getting back to me. I'm using v2 and Office 365 although the issue shows on all Excel versions as far as I know.

Please see attached worksheet with a chart showing the negative axis

[ExcelSize-1.xlsx](https://github.com/qax-os/excelize/files/8442881/ExcelSize-1.xlsx)

**Comment by xuri** [comment]
Thanks for your feedback. I'll fix this issue in the next released version. Next release is planned for Monday, April 11th, 2022.

**Comment by xuri** [comment, marked as answer]
Please upgrade to the new version 2.6.0, and specify the minimum as 0 for the Y-axis in the chart parameter like this:

```go
if err := f.AddChart("ActLog_IOSTAT_IOPS_Interval", "C35", `{
    "type": "line",
    "series": [
    {
        "name": "ActLog_IOSTAT_IOPS_Interval!$A$33",
        "categories": "ActLog_IOSTAT_IOPS_Interval!$B$32$CNL$32",
        "values": "ActLog_IOSTAT_IOPS_Interval!$B$33:$CNL$33",
        "marker":
        {
            "symbol": "none"
        }
    }],
    "title":
    {
        "name": "Disk Activity by Interval"
    },
    "format":
    {
        "x_scale": 2,
        "y_scale": 2
    },
    "y_axis":
    {
        "minimum": 0
    }
}`); err != nil {
    fmt.Println(err)
    return
}
```
